src/liquid/components/ld-radio/ld-radio.tsx

```diff
@@ -50,25 +66,44 @@ export class LdRadio implements InnerFocusable {
   }
 
   @Watch('checked')
+  @Watch('form')
   @Watch('name')
   @Watch('required')
   @Watch('value')
   updateHiddenInput() {
+    const outerForm = this.el.closest('form')
+    if (!this.hiddenInput && this.name && (outerForm || this.form)) {
+      this.hiddenInput = document.createElement('input')
```

Comment:
Same as for checkbox. (type/style)
```suggestion
      this.hiddenInput = document.createElement('input')
      this.hiddenInput.type = 'radio'
      this.hiddenInput.style.visibility = 'hidden'
      this.hiddenInput.style.position = 'absolute'
      this.hiddenInput.style.pointerEvents = 'none'
```